sonar-ws/src/main/java/org/sonarqube/ws/client/issue/IssueFilterParameters.java

```diff
@@ -53,6 +53,11 @@
    */
   @Deprecated
   public static final String DEPRECATED_ACTION_PLANS = "actionPlans";
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Do not forget to remove this deprecated code someday. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/coding_rules#rule_key=squid%3AS1133)